packaging/installer/netdata-updater.sh

```diff
@@ -193,6 +201,11 @@ update() {
   return 0
 }
 
+logfile=
+tmpdir=
+
+trap cleanup 0 1 2 3 6
```

Comment:
How about `trap cleanup EXIT`?

Reply:
Okay :)

The only documentation I can find on this `EXIT` is this:

> The most common use of the trap command though is to trap the bash-generated psuedo-signal named EXIT. Say, for example, that you have a script that creates a temporary file. Rather than deleting it at each place where you exit your script, you just put a trap command at the start of your script that deletes the file on exit:

Reply:
The benefit to trapping on `EXIT` is that it catches all the signals you could trap on, and also traps on most other ways of exiting the script that aren't signal-based. In this case it's just a bit more thorough. And yes, it's not very well documented.